src/Illuminate/Foundation/Application.php

```diff
@@ -552,6 +558,15 @@ protected function markAsRegistered($provider)
 		$this->serviceProviders[] = $provider;
 
 		$this->loadedProviders[$class] = true;
+		
+		$aliases = class_parents($class) + class_implements($class) + array($class);
```

Comment:
So you're saying you would prefer:

```php
foreach (class_parents($class) + class_implements($class) + array($class) as $alias)
```
(which is 84 characters long, not counting indentation of 8.)

Is that right?  I'm not sure it would improve readability or performance, but I don't mind changing it I guess.